utils/cli/core/core.py

```diff
@@ -182,8 +182,13 @@ class CVAT_API_V1():
     """ Build parameterized API URLs """
 
     def __init__(self, host, https=False):
```

Comment:
You may drop the 'https' argument from function declaration, aren't you?

Reply:
There is a parameter in CLI to enforce https, which uses this function parameter.